tests/_support/Step/Acceptance/PaymentMethod/GenericPaymentMethodStep.php

```diff
@@ -26,7 +26,8 @@ class GenericPaymentMethodStep extends GenericStep
     private $configObjectMap = [
         self::CREDIT_CARD => CreditCardConfig::class,
         self::CREDIT_CARD_ONE_CLICK => CreditCardConfig::class,
-        self::PAY_PAL => PayPalConfig::class
+        self::PAY_PAL => PayPalConfig::class,
+        self::IDEAL => PayPalConfig::class
```

Comment:
Does this work with PayPalConfig? Maybe it's better to create an iDeal config class that just extends PayPalConfig to be cleaner?

Reply:
Test-case works like a charm. I totally agree it that would be cleaner to create PayPalConfig class and PayPalData.json file, but since these are not necessary for this test-case (there is nothing to put in json file, so class would not have a single line of code), and paymentMethod instance cannot be created without these files, I decided to make this workaround.

Reply:
ok, I see. I'm fine with this for this US then

Reply:
Could you please tell me where can I find documentation for iDealConfig.json? I couldn't find it, so I reused entries for CreditCard and updated values.
I'm also wondering if entries I added to WoocommerceMappedPaymentActions.json file are correct.

Reply:
Here is the documentation https://doc.wirecard.com/iDEAL_Main.html in Test Credentials section

Reply:
Also as a tip  to see the fields that are needed I usually configure payment method manually on the clean install and look into the database to see which fields were added

Reply:
Thanks. Good to know that for next test-cases.